docs/integrations/microsoft-azure/azure-database-for-postgresql.md

```diff
@@ -0,0 +1,69 @@
+---
+id: azure-database-for-postgresql
+title: Azure Database for PostgreSQL
+description: Learn about the Sumo Logic collection process for the Azure Database for PostgreSQL service.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/azure-database-for-postgresql.png')} alt="Thumbnail icon" width="50"/>
+
+[Azure Database for PostgreSQL](https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/overview) is a fully managed relational database service in the Microsoft cloud based on the PostgreSQL community edition. This integration helps in monitoring resource utilization and identifying slow queries to optimize your workloads and configure your server for the best performance.
+
+The below instructions applies to Azure Database for PostgreSQL - Flexible Server as Azure Database for PostgreSQL - Single Server is on the retirement path.
```

Comment:
Can you be more clear here, it is confusing with two hypens.

Reply:
so basically Azure Database for PostgreSQL - Flexible Server and Azure Database for PostgreSQL - Single Server are separate configurations for this database.

Reply:
changing this to "applies to Azure Database for PostgreSQL with Flexible Server only."